index.html

```diff
@@ -240,15 +240,29 @@ <h1>Biometrics</h1>
   <div class="content">
     <div class="row">
     <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3>
-    <p> All cameras in Chicago's network may have still shots captured from them to use for identification purposes. </p>
+    <p> CPD has released very little information as to how they use FRT, although it seems that most of the infrastructure and personnel involved with FRT are located in the Crime Prevention and Information Center (CPIC), located in the Public Safety Headquarters. Still images from <a href="#cameras">cameras</a> in Chicago's camera network can be analyzed with FRT. </p>
```

Comment:
I think it would be good to talk about fusion centers in general and what CPIC is. I can send you language tomorrow am.